src/acceptance/app/dynamic_policy_test.go

```diff
@@ -215,7 +215,7 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 							return
 						case <-ticker.C:
 							Eventually(func() string {
-								return helpers.CurlAppWithTimeout(cfg, appName, "/fast", 10*time.Second)
+								return helpers.CurlAppWithTimeout(cfg, appName, "/fast", 30*time.Second)
```

Comment:
Where waiting time in Eventually is 10 sec,  does it make sense to wait for the curl operation inside for 30 sec ?